src/order_cmd.cpp

```diff
@@ -2020,7 +2019,8 @@ bool UpdateOrderDest(Vehicle *v, const Order *order, int conditional_depth, bool
 				} else {
 					Aircraft *a = Aircraft::From(v);
 					DestinationID destination = a->current_order.GetDestination();
-					if (a->targetairport != destination) {
+					Depot *hangar = Station::Get(a->targetairport)->airport.hangar;
```

Comment:
Should this be `Station::GetIfValid()` since the next line checks if `hanger != nullptr`?